<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GerirCarro</class>
 <widget class="QWidget" name="GerirCarro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerir Carro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon.ico</normaloff>../icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <widget class="QCommandLinkButton" name="commandButtonVoltar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../setavoltar.ico</normaloff>../setavoltar.ico</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Carro:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewCarro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewModulosCarro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>761</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Módulos deste Carro:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>791</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnApagarCarro">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Apagar Carro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardarCarro">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCriarModulo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Criar Módulo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomeCarro">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nota: Fazer double-lick na linha respetiva ao módulo que pretende ver os dados a pormenor.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
